### DISCUSSION TITLE
Phishing mail going around.

### DISCUSSION THREAD

**Discussion by xchng** [community/community#41201] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I'd like to notify of a phishing mail going around: 
![image](https://user-images.githubusercontent.com/4209653/206730663-69e45eaf-017a-4386-9e3e-9fb53a45b2d5.png)

The button leads to an url https://gjthub.net/login [FAKE: Notice the j instead of i]
Sender was info@mywellnesscheck.org

I hope this helps.

**Comment by JJJ** [comment]
Me, too!

Seemed legjt, so J’ve already moved all my repos to gjt & GjtHub jn support of thjs obvjously excellent new jnjtjatjve 💙

Stay vjgjlant, y’all! 🦙

**Reply by dermoth** [reply]
Indeed - I even missed the `j` when I checked the link, and I had recently logged from a new computer so it all seemed very legit. Fortunately I also never click on links from unsolicited emails like these and rather login from a known url. Even the path seemed legit but then trying it from the github.com domain led to a 404, and this is where I finally realized the typo!

So I'm glad my 2nd line of defense saved me, but it's by far the best phishing attempt I've ever received.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @xchng 👋🏼  We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you!
